app/controllers/data_exports_controller.rb

```diff
@@ -2,12 +2,24 @@
 
 # Controller actions for Data Exports
 class DataExportsController < ApplicationController
-  before_action :data_export, only: %i[download destroy]
+  before_action :data_export, only: %i[download destroy show remove]
   before_action :data_exports, only: %i[index destroy]
   before_action :current_page
 
   def index; end
 
+  def show
+    authorize! @data_export, to: :read_export?
+
+    # Avoids manually navigating to an empty manifest page
+    if @data_export.manifest.empty?
+      @tab = 'summary'
+    else
+      @tab = params[:tab]
```

Comment:
**nitpick (blocking):** Instead of setting  `@tab` here can you update your logic to match what I did here https://github.com/phac-nml/irida-next/pull/512
Look at `workflow_executions_controller.rb` and specifically `set_default_tab`.

Reply:
Fixed in 0cce62cbcf934784a58a3d9264a98792162aa526